### PR TITLE
feat:added wishlist page

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/154797423-6fc40344-d47a-43da-8061-8b8e6e02b2fc.mp4

### CODE REVIEW

File: wishlist page/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HomePage</title>
+    <link rel="stylesheet" href="/homepage/homepage.css">
+    <link rel="stylesheet" href="/product listing page/products.css">
+    <link rel="stylesheet" href="wishlist.css"> <link rel="stylesheet" href="../css/base.css">
+    <link rel="stylesheet" href="../css/spacing.css">
+    <link rel="stylesheet" href="https://galaxia-designs.netlify.app/styles.css" />
+    <script src="https://kit.fontawesome.com/7f4b9645a7.js" crossorigin="anonymous"></script>
```
Comment (Hamdrive) [inline wishlist page/wishlist.html:13]: Consider importing this directly into your main CSS file with the CDN link, thus you wont have to keep importing it for each HTML file

File: wishlist page/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+                        <input type="search" placeholder=" Search..." name="search" class="icon-search"></form>
+                </div>
+                <div class="right-links">
+                    <ul>
+                        <li><a href=""><i class="fa-solid fa-heart wish-list padding-i color"></i></a>
+                        </li>
+                        <li><a href=""><i class="fa-solid fa-cart-shopping padding-i"></i></a>
+                        </li>
+                        <li <a href="" class="log-in padding-i">LOGIN</a></li>
```
Comment (Hamdrive) [inline wishlist page/wishlist.html]: If these icons are not linked to any pages as of yet, then give it a placeholder `#`

**Review by Hamdrive — APPROVED** [review]
Good work done, have suggested some changes which can be committed easily.
Else all looks good and can be merged!

File: wishlist page/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.wishlist-container{
+    width: 80%;
+    display: flex;
+    height: auto;
+    flex-wrap: wrap;
```
Comment (nareshpoonia) [inline wishlist page/wishlist.css:5]: You'd be using display:flex and flex-wrap: wrap in many projects, you can consider making utility classes for these.

**Review by nareshpoonia — APPROVED** [review]
Just a small comment on using utility classes, other than that code looks good and clean.
Good to merge

File: wishlist page/wishlist.css (hunk 3 above)
Comment (stutisk) [reply wishlist page/wishlist.css:5]: yes you are right will make another branch to fix this one

**Pull request merged**